src/mero_cluster.erl

```diff
@@ -119,6 +119,26 @@
               {mero_cluster_util, worker_by_index, 3},
               {mero_cluster_util, module_info, 1}]).
 
+-if(?OTP_RELEASE >= 27).
+
+-dialyzer({[no_unknown],
```

Comment:
Can you add a comment explaining this dialyzer config?

Reply:
There it is.